en/docs/install-and-setup/setup/mi-setup/deployment/mi-cicd-k8s.md

```diff
@@ -0,0 +1,134 @@
+## Overview
+
+[![K8s based CI/CD for Micro Integrator]({{base_path}}/assets/img/deploy/mi-cicd-K8s.png)]({{base_path}}/assets/img/deploy/mi-cicd-K8s.png)
+
+#### Overview
+
+There are two kinds of jobs in Jenkins that we need to maintain. One for the Integration project repositories. Another kind is for the Deployment Descriptor repositories. 
+
+#### Integration Project Build Job
+- We need to maintain one Jenkins job per Integration Project repository.
+- The Integration Project has to be a [Maven Multi Module project]({{base_path}}/integrate/develop/create-integration-project/#maven-multi-module-projects) and it has to contain one Kubernetes Exporter module. 
+- The build phase of the job will build the Integration project and run the unit tests if a Unit test server, if configured.
+- The release phase of the job generates docker images using the provided repository, name and project version and pushes the image to the configured docker registry and creates a release tag in github.
```

Comment:
```suggestion
- The release phase of the job generates Docker images using the provided repository, name, and project version and pushes the image to the configured Docker registry and creates a release tag in GitHub.
```
Please change "docker" to "Docker" and "github" or other variants to "GitHub" in all places.

Reply:
Done